Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/TicketService.hs

```diff
@@ -293,6 +293,14 @@ postTicketPlacesBook (mbPersonId, merchantId) placeId req = do
       now <- getCurrentTime
       ticketService <- QTS.findById ticketServicesReq.serviceId >>= fromMaybeM (TicketServiceNotFound ticketServicesReq.serviceId.getId)
       businessHour <- QBH.findById bHourId >>= fromMaybeM (BusinessHourNotFound bHourId.getId)
+
+      let startTime = case businessHour.btype of
+            Domain.Types.BusinessHour.Slot time -> time
+            Domain.Types.BusinessHour.Duration startTime' _ -> startTime'
+      let visitDateTime = UTCTime visitDate (timeOfDayToTime startTime)
+      
+      when (visitDateTime < now) $ throwError $ InvalidRequest "Cannot book for past date"
```

Comment:
Can we allow till some time of start of the show? For example, when we are late for movies, we get tickets from counter but not online, we can allow online bookings too for some period(10 mis let's say?). Any issue with that @sidsethu?